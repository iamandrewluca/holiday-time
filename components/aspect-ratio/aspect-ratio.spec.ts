import { describe, it, expect } from '@jest/globals'
import { render } from '@testing-library/vue'
import AspectRatio from './aspect-ratio.component.vue'

describe('AspectRatio', () => {
  it('should render and match snapshot', () => {
    const { baseElement } = render(AspectRatio, {
      props: { h: 1, w: 1 },
    })
    expect(baseElement).toMatchSnapshot()
  })
})
