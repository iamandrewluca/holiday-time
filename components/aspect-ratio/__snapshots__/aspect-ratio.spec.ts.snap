// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AspectRatio should render and match snapshot 1`] = `
<body>
  <div>
    <div
      class="root"
    >
      <div
        class="inner"
      />
    </div>
  </div>
</body>
`;
